charts/operator/README.md

```diff
@@ -23,7 +23,7 @@ This chart installs [Rookout's k8s Operator](https://docs.rookout.com/docs/k8s-o
 To install the chart with the release name `my-release`:
 
 ```bash
-$ helm install --name my-operator rookout/operator --set operator.token=YOUR_ORGANIZATIONAL_TOKEN
+$ helm install my-operator rookout/operator --set operator.token=YOUR_ORGANIZATIONAL_TOKEN
```

Comment:
I think that `--name` is for helm 2 only, so I understand this is going to apply for helm 3 only.
What's the reason the whole chart wont work for helm2?

Reply:
the `--set installCRDs=true` not works for helm 2.

Reply:
Oh I remember now, you said that there the helm2 can't reload the crds first.
I googled it and found out that you need to use the `crd-install` hooks

Look at method 2 in:
https://v2.helm.sh/docs/chart_best_practices/#install-a-crd-declaration-before-using-the-resource

Reply:
adding the hook might not work for helm3, so we can 2 options:
1. Have 2 folders, for each version of helm
2. Add to the readme that helm2 users need to run the CRDs first before running the `helm install`, it would warn them that the CRD is already installed when they run the `helm install`, but that's fine.